steam-market-seller.user.js

```diff
@@ -439,7 +474,9 @@
         };
 
         items.forEach(function(item) {
-            itemQueue.push(item);
+            setTimeout(function(){
+                itemQueue.push(item);
+            }, 500); // have some healthy delay or steam will block you for flooding
```

Comment:
I start from recent fork from @Tackyou and I didn't check that code.
Script sells items one by one and that delay is not needed.
I push a new commit to remove that delay.